view/desktop/src/components/PageLoader.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
+
+const tips = [
+  "The statusbar color can be changed in the appearance settings",
+  "You can change the order of widget actions. Try it!",
+  "Lorem ipsum dolor sit amat. Met the statusbar color can be changed in the appearance settings",
+];
```

Comment:
Make sure to mark temporary solutions in the code with explicit comments so that new team members and other developers understand that a certain piece of code is a temporary workaround and will be replaced with a specific implementation in the future.

For example, in this case, you could write:

```typescript
// The statically created array `tips` is a temporary solution.
// In the future, this data will be fetched from the backend.
```